client/my-sites/stats/summary/index.jsx

```diff
@@ -92,6 +96,28 @@ class StatsSummary extends Component {
 		);
 	}
 
+	getPath( statType ) {
+		if ( statType === 'statsCountryViews' ) {
+			const geoMode = this.props.context.query.geoMode;
+			const geoModeLabel =
+				geoMode && Object.prototype.hasOwnProperty.call( GEO_MODES, geoMode )
+					? GEO_MODES[ geoMode ]
+					: 'country';
+
+			return `locations-${ geoModeLabel }`;
```

Comment:
```suggestion
			return `${ path }-${ geoModeLabel }`;
```
There is still another `countryviews` path mapping to the `statsCountryViews` statType.